python/ray/serve/scripts.py

```diff
@@ -330,7 +330,7 @@ def run(
                 handle = serve.get_deployment("DAGDriver").get_handle()
         else:
             handle = serve.run(node, host=host, port=port)
-        cli_logger.success("Deployed successfully.")
+        cli_logger.success("Submitted deploy config successfully.")
```

Comment:
Could we create two separate `cli_logger` statements, one for the `if` condition and one for the `else` condition? When the user deploys with the config (the `if` clause), I think we can say

```python
cli_logger.success("Submitted Serve config successfully.")
```

When the user deploys with an import path (the `else` clause), I think we can leave it as:

```python
cli_logger.success("Deployed Serve app successfully.")
```